docs/sources/flow/reference/components/prometheus.exporter.windows.md

```diff
@@ -0,0 +1,223 @@
+[windows_exporter](https://github.com/prometheus-community/windows_exporter) which exposes a
+wide variety of hardware and OS metrics for Windows-based systems.
+
+The `windows_exporter` itself comprises various _collectors_, which can be
+enabled and disabled at will. For more information on collectors, refer to the
+[`collectors-list`](#collectors-list) section.
+
+## Usage
+
+```river
+prometheus.exporter.windows "LABEL" { 
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+| Name                       | Type           | Description                                                   | Default | Required |
+|----------------------------|----------------|---------------------------------------------------------------|---------|----------|
+| `enabled_collectors`       | string`        | List of collectors to enable.                                |         | no       |
+
+`enabled_collectors` defines a hand-picked list of enabled-by-default
+collectors. If set, anything not provided in that list is disabled by
+default. See the [Collectors list](#collectors-list) for the default set.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`prometheus.exporter.windows` to configure collector-specific options:
+
+Hierarchy    | Name             | Description                            | Required 
+-------------|------------------|----------------------------------------|----------
+exchange     | [exchange][]     | Configures the exchange collector.     | no       
+iis          | [iis][]          | Configures the iis collector.          | no       
+text_file    | [text_file][]    | Configures the text_file collector.    | no       
+smtp         | [smtp][]         | Configures the smtp collector.         | no       
+service      | [service][]      | Configures the service collector.      | no       
+process      | [process][]      | Configures the process collector.      | no       
+network      | [network][]      | Configures the network collector.      | no       
+mssql        | [mssql][]        | Configures the mssql collector.        | no       
+msmq         | [msmq][]         | Configures the msmq collector.         | no       
+logical_disk | [logical_disk][] | Configures the logical_disk collector. | no       
+
+[exchange]: #exchange-block
+[iis]: #iis-block
+[text_file]: #textfile-block
+[smtp]: #smtp-block
+[service]: #service-block
+[process]: #process-block
+[network]: #network-block
+[mssql]: #mssql-block
+[msmq]: #msmq-block
+[logical_disk]: #logicaldisk-block
+
+### exchange block
+Name | Type     | Description | Default | Required
+---- |----------| ----------- | ------- | --------
+`enabled_list` | `string` | Comma-separated list of collectors to use, for example:`AvailabilityService,OutlookWebAccess`. Matching is case-sensetive |  | no
```

Comment:
Can you move the examples outside of the table? Otherwise the table doesn't render very well on the docs site:

```suggestion
Name | Type     | Description | Default | Required
---- |----------| ----------- | ------- | --------
`enabled_list` | `string` | Comma-separated list of collectors to use. | | no

The collectors specified by `enabled_list` can include the following:

- `ADAccessProcesses`
- `TransportQueues`
- `HttpProxy`
- `ActiveSync`
- `AvailabilityService`
- `OutlookWebAccess`
- `Autodiscover`
- `WorkloadManagement`
- `RpcClientAccess` 

For example, `enabled_list` may be set to `"AvailabilityService,OutlookWebAccess"`. 

```